src/java.base/share/classes/java/io/InputStream.java

```diff
@@ -53,8 +54,23 @@ public abstract class InputStream implements Closeable {
     // use when skipping.
     private static final int MAX_SKIP_BUFFER_SIZE = 2048;
 
+    private static final int MIN_SKIP_BUFFER_SIZE = 128;
```

Comment:
@jmehrens mentioned that this one can be removed as well since it's not used.

Reply:
@jmehrens @liach you are correct. done.